internal/cliwrapper/cliwrapper.go

```diff
@@ -15,12 +15,19 @@ import (
 type cliWrapper struct {
 	podmanFullPath string
 	logger         log.Logger
+	runtimeContext []string
```

Comment:
Why `runtimeContext` ... that seems really vague and generic when what you're specifying is a remote podman "connection" name.

Reply:
Making it generic to keep the same name in the docker deployer which has a similar command: https://docs.docker.com/engine/context/working-with-contexts/#use-a-different-context

Reply:
So the question is then can the same type of input be used for both podman and docker?

Reply:
Ah: and Podman seems to support `podman context` as well as `podman system connection`. Still, I think "runtime context" isn't obviously the same, and sounds a lot less specific. If you want to hold onto the Docker "context" perhaps try to "connect" that to the Podman "connection" term as something like `ConnectionContext` (which actually gives some clue to what it does).

And as hinted by Jared's comment, we need to think about whether to extend the podman deployer support to the generalized "connection/context" mechanism without doing the same for the docker deployer. It should definitely look the same to the user.

Reply:
Did not realize podman had a context option, looks like they are aliases for podman connection based on this https://github.com/containers/podman/issues/12811#issuecomment-1153128175. Could get rid of `connection` and use `context` so it applies to both.

Reply:
Although I haven't looked into all the details, I believe that the original docker "context" (they don't reference "runtime context", which I still think implies something very different) deals with mapping remote sockets to a "context name", a capability that podman (I believe in 4.3) adopted as an extension to their "connection" mechanism which allowed specifying a REST API socket, including using designated SSH RSA tokens, which seems to go a bit beyond docker; while the podman "context" models the docker CLI syntax and accepts (but apparently doesn't implement) some of the options...

So basically "docker context" and "podman context" allow specifying a remote port, including ssh, tcp, or unix sockets, while podman "connections" add ssh RSA tokens for ID.

Reply:
I tried using the `--context` option on podman and it seems half baked. It doesn't allow you to add a SSH key so I plan on making this a podman only option and make the variables more specific.

Reply:
Tracking it in this thread instead: https://github.com/arcalot/arcaflow-engine-deployer-podman/pull/61#discussion_r1581211823